pkg/recipes/driver/bicep.go

```diff
@@ -362,6 +362,12 @@ func (d *bicepDriver) prepareRecipeResponse(outputs any, resources []*armresourc
 		}
 	}
 
+	recipeResponse.Status = &rpv1.RecipeStatus{
+		TemplateKind:    recipes.TemplateKindBicep,
+		TemplatePath:    templatePath,
+		TemplateVersion: "",
```

Comment:
Do we need to add the `TemplateVersion` as empty or can we skip setting it here? It would be good to show template version field in the response only for scenarios where it makes sense.